image_classification/MAE/configs/vit_base_patch16_224_pretrain_dec1.yaml

```diff
@@ -0,0 +1,36 @@
+DATA:
+    IMAGE_SIZE: 224
+    CROP_PCT: 0.875
+MODEL:
+    TYPE: MAE
+    NAME: vit_base_patch16_224_dec1
+    DROPPATH: 0.0
+    TRANS:
+        PATCH_SIZE: 16
+        MLP_RATIO: 4.0
+        QKV_BIAS: true
+        MASK_RATIO: 0.75
+        ENCODER:
+            EMBED_DIM: 768
+            DEPTH: 12
+            NUM_HEADS: 12
+        DECODER:
+            EMBED_DIM: 512
+            DEPTH: 1
+            NUM_HEADS: 8
+TRAIN:
+    NUM_EPOCHS: 800
+    WARMUP_EPOCHS: 40
+    WEIGHT_DECAY: 0.05
+    BASE_LR: 0.75e-4 #1.5e-4
+    WARMUP_START_LR: 1e-6
+    GRAD_CLIP: 5.0
```

Comment:
Just to remind you, the original paper doesn't use gradient clip.

Reply:
Now the training still have NaN error after dozens of epochs, still trying these hyperparams.